kolibri/plugins/learn/assets/src/views/HybridLearningContentCardListView.vue

```diff
@@ -131,16 +158,6 @@
       now: now(),
     }),
     computed: {
-      categoryAndLevelString() {
```

Comment:
Reason for changing this from a computed prop? I still only see it being used for `this.content`?

Reply:
no good reason! will revert